doc/deprecated.rst

```diff
@@ -223,3 +223,17 @@ Testing Utilities
 
 * The data providers ``getTests()`` and ``getLegacyTests()`` on
   ``Twig\Test\IntegrationTestCase`` are considered final als of Twig 3.13.
+
+Environment
+-----------
+
+* The ``Twig\Environment::mergeGlobals()`` method is deprecated as of Twig 3.13
+  and will be removed in Twig 4.0:
+
+  Before::
+
+      $context = $twig->mergeGlobals($context);
+
+  After::
+
+      $context += $twig->getGlobals();
```

Comment:
`getGlobals()` is flagged as `@internal`. Shall we remove that annotation?

Reply:
But then it doesn't make sense to document that method as a replacement for a deprecated public method.

@fabpot Can you please clarify how the deprecation should be resolved downstream?

Reply:
Making `getGlobals()` `private` has been reverted.

Reply:
@xabbuh I’m sorry, you’re right! I didn’t notice the direct commit in v4. But then @derrabus question is still valid: shouldn’t be the flag removed?